pkg/kube/controllers/extendedjob/job_creator.go

```diff
@@ -103,13 +103,19 @@ func (j jobCreatorImpl) Create(ctx context.Context, eJob ejv1.ExtendedJob, podNa
 	if err != nil {
 		return false, errors.Wrapf(err, "could not generate job name for eJob '%s'", eJob.Name)
 	}
+
+	backoffLimit := int32(2)
```

Comment:
Use `Int32()` in `pkg/kube/util/pointer.go` is better.

Reply:
Done.